pyvo/dal/mimetype.py

```diff
@@ -75,32 +75,37 @@ def mime_object_maker(url, mimetype, session=None):
     if not mimetype:
         raise ValueError('mimetype required')
     session = use_session(session)
-    full_type, params = cgi.parse_header(mimetype)
-    mimetype = [x.strip() for x in full_type.split('/')] if '/' in full_type \
+    m = Message()
+    m['content-type'] = mimetype
+    pp = m.get_params()
```

Comment:
I prefer meaningful variable names, but as these are not really used later on, I suppose they could stay as is